webapp/src/ts/services/user-settings.service.ts

```diff
@@ -60,6 +62,22 @@ export class UserSettingsService {
     });
   }
 
+  async hasMultipleFacilities(): Promise<boolean> {
+    return this.get()
+      .then((userSettings:any) => {
+        const userFacility = userSettings.facility_id;
+        return Array.isArray(userFacility) && userFacility.length > 1;
+      });
```

Comment:
```suggestion
    return this
      .get()
      .then(userSettings => {
        const userFacility = userSettings.facility_id;
        return Array.isArray(userFacility) && userFacility.length > 1;
      });
```

Reply:
We avoid `any` as we're working toward defining the data for clarity. Instead, define a type